README.ctan

```diff
@@ -0,0 +1,49 @@
+fonts-churchslavonic
+=====================
+
+Collection of free fonts for Church Slavonic language (also known as Church Slavic
+and Old Church Slavonic), ISO 639-1 code "cu".
+
+All fonts use Unicode character encoding and are intended for Unicode TeX engines
+only.
+
+
+Installation:
+-------------
+
+	tlmgr install fonts-churchslavonic
+
+
+Using fonts
+-----------
+
+Use "fontspec" LaTeX package to load any of the installed fonts. For example:
+
+	\newfontfamily\ponomar[Script=Cyrillic,HyphenChar=_]{Ponomar Unicode}
```

Comment:
Fonts installed by tlmgr can not be loaded by name (unless also installed on system level). Lets change this to "PonomarUnicode.otf".